scripts/deploy.sh

```diff
@@ -99,6 +99,8 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   then
     rm /etc/openstack_deploy/conf.d/swift.yml
```

Comment:
`rm -f` for consistency...?...even though the `-f` is technically not needed anymore, since these files are being added as part of the AIO host-bootstrap role...